src/components/backup/BackupIcloudStep.js

```diff
@@ -216,10 +218,10 @@ const BackupIcloudStep = () => {
           newLabel = '💩 Weak password';
           break;
         case 2:
-          newLabel = '💪 Good password';
+          newLabel = '👌 Good password';
           break;
         case 3:
-          newLabel = '🦾 Great password';
+          newLabel = '💪 Great password';
```

Comment:
Just FYI: The emojis I put here were the ones in the Figma designs.

Reply:
Cool, yeah mentioned it to @christianbaroni - gonna merge in this PR for now, lmk if you want it to be something else!